github.go

```diff
@@ -147,3 +174,80 @@ func existRelease(ctx context.Context, client *github.Client, owner, repo, tag s
 	}
 	return resp.StatusCode == http.StatusOK, nil
 }
+
+func (g *githubClient) getPullRequest(ctx context.Context, owner, repo string, number int) (*github.PullRequest, error) {
+	pr, _, err := g.restClient.PullRequests.Get(ctx, owner, repo, number)
+	return pr, err
+}
+
+type (
+	PullRequestState     string
+	PullRequestSort      string
+	PullRequestDirection string
+)
+
+const (
+	// PullRequestState
+	PullRequestStateOpen   PullRequestState = "open"
+	PullRequestStateClosed PullRequestState = "closed"
+	PullRequestStateAll    PullRequestState = "all"
+
+	// PullRequestSort
+	PullRequestSortCreated     PullRequestSort = "created"
+	PullRequestSortUpdated     PullRequestSort = "updated"
+	PullRequestSortPopularity  PullRequestSort = "popularity"
+	PullRequestSortLongRunning PullRequestSort = "long-running"
+
+	// PullRequestDirection
+	PullRequestDirectionAsc  PullRequestDirection = "asc"
+	PullRequestDirectionDesc PullRequestDirection = "desc"
+)
+
+func (p PullRequestState) String() string {
+	return string(p)
+}
+
+func (p PullRequestSort) String() string {
+	return string(p)
+}
+
+func (p PullRequestDirection) String() string {
+	return string(p)
+}
+
+type ListPullRequestOptions struct {
+	State     PullRequestState
+	Sort      PullRequestSort
+	Direction PullRequestDirection
+	Limit     int
+}
+
+func (g *githubClient) listPullRequests(ctx context.Context, owner, repo string, opt *ListPullRequestOptions) ([]*github.PullRequest, error) {
+	const perPage = 100
```

Comment:
> Results per page (max 100)

https://docs.github.com/en/rest/reference/pulls#list-pull-requests

Reply:
Got it. 👍